src/meltano/core/block/extract_load.py

```diff
@@ -535,11 +535,14 @@ async def run_with_job(self) -> None:
                 self.context.session,
             )
         ):
-            raise RunnerError(
-                f"Another '{job.job_name}' pipeline is already running "  # noqa: EM102
-                f"which started at {existing.started_at}. To ignore this "
-                "check use the '--force' option.",
+            msg = (
+                f"Another '{job.job_name}' pipeline is already running "
+                f"which started at {existing.started_at}. "
+                f"Wait until {existing.valid_intil()} when the job will be "
```

Comment:
Will this print it in the form `datetime.datetime(...)`? We should probably call a method on `existing.valid_intil()` to convert it to a string formatted in the way we want. Maybe `existing.valid_intil().isoformat()` is suitable here.